content/docs/user-guide/components/reference/atom/diffuse-gi.md

```diff
@@ -7,6 +7,10 @@ toc: true
 
 The **Diffuse Global Illumination** component is a Level component that controls the quality level of global illumination that **Diffuse Probe Grid** components provide. The diffuse global illumination technique calculates the effect of light bounce, reflection, scatter, and absorption throughout an environment. It simulates real-world lighting behavior, where objects are lit both from the light source and from the light reflected by other objects.
 
+{{< note >}}
+You must have Shader Model 6.3 or later to use raytracing in the **Atom Renderer**.
```

Comment:
As another reviewer commented, perhaps "having Shader Model 6.3" could be reworded to better define what we mean. Would it be more accurate, and sufficient, to say "Your GPU must support Shader Model 6.3" to use raytracing? If so, consider linking to [https://en.wikipedia.org/wiki/High-Level_Shading_Language](https://en.wikipedia.org/wiki/High-Level_Shading_Language). Or do we also need to mention graphic driver version support?

Reply:
```suggestion
You must have a GPU with support for DirectX 12 Shader Model 6.3 or later to use raytracing in the **Atom Renderer**.
```

Reply:
I'll change the wording to the one Stephen suggested. 

I'm not sure if we need to link to that wiki, since it doesn't exactly point to the Shader Model 6.3. It's moreso deeper information that customers can search themselves if they are interested in what Shader Models are. But for the most part, they just need to be aware that their GPU supports Shader Model 6.3.